apps/server/src/modules/h5p-editor/controller/dto/h5p-editor.params.ts

```diff
@@ -16,6 +16,20 @@ export class GetH5PContentParams {
 	contentId!: string;
 }
 
+export class GetH5PEditorParams {
+	@ApiProperty()
+	@Matches('([a-z]+-[a-z]+)')
+	@IsString()
+	@IsOptional()
+	language?: string;
+
+	@ApiProperty()
+	@Matches('([A-Z0-9a-z]+)')
+	@IsString()
+	@IsOptional()
+	contentId?: string;
```

Comment:
can you improve validation? ids are usually a strict format, and so, validation is for that format only
uuid, mongoid, a fixed lenght  etc
also, always escape strings with SanitizeHtml() decodator

Reply:
IDs are currently generated by lumi, we will evaluate if it is possible to switch them out for UUIDs or similar. As this is not a part of this PR we will add this to the backlog